charts/cbcontainers-agent/README.md

```diff
@@ -0,0 +1,43 @@
+# cbcontainers-agent
+
+This is the official Helm chart for installation of the CBContainers agent.
+
+## Installation
+
+In order for the chart to be installed it needs minimal configuration.
+
+There are 8 required fields that need to be provided by the user:
+
```

Comment:
Maybe add a line describing which version of Helm is supported (2, 3 or both) since 2 and 3 have major differences AFAIK